standard/documentation-comments.md

```diff
@@ -1031,8 +1031,8 @@ namespace Graphics
         /// This constructor initializes the new Point to
         /// (<paramref name="xPosition"/>,<paramref name="yPosition"/>).
         /// </summary>
-        /// <param><c>xPosition</c> is the new Point's x-coordinate.</param>
-        /// <param><c>yPosition</c> is the new Point's y-coordinate.</param>
+        /// <param name="xPosition"><c>xPosition</c> is the new Point's x-coordinate.</param>
```

Comment:
At some point we should improve the docs to remove the "xPosition is..." bits. But not now :)

Reply:
I had wanted to do it anyway. Thanks for pointing that out.

Reply:
I went ahead and just did it.